ChaseBENNC9/GameJam-Charika
# Checkpoints

This feature will add checkpoints into the game. Which means after integrating the security lights and any other hazard the player won't have to start the level from the very beginning each time. 

# How to test
1 - Open  checkpoints scene
2 - press r once. the player should reset to the start of the level
3 - jump over the first wall. and move forwards a bit
4 - press r again , this time the player will reset to the other side of the wall.
